### DISCUSSION TITLE
Teleport Immunity message

### DISCUSSION THREAD

**Discussion by Chivitos100** [EssentialsX/Essentials#3990] [Ideas and Feature Suggestions] [answered]
Hello, I am chivitos. My request today will be a teleport immunity message.

**How would it contribute?**
When there is teleport immunity, nothing is displayed, so it's kind of a problem since users confuse a lot because they cannot pvp.

**What do I exactly suggest?**
A message that gets displayed when someone tries to punch another user which has their teleport immunity active.

**Example:** 
English: &4You cannot punch this player yet. They still have teleport immunity on for &c20 Seconds&4.
Spanish: &4No puedes golpear a este jugador aun. Su inmunidad de teletransporte esta activo aun por &c20 Segundos&4.

**Comment by pop4959** [comment, marked as answer]
I can see how that can be confusing, but this is probably better handled by a dedicated PvP plugin rather than in Essentials itself, as that's not really what the teleportation immunity feature was intended for.
